service/src/test/java/bio/terra/workspace/pact/WdsContractVerificationTest.java

```diff
@@ -0,0 +1,133 @@
+package bio.terra.workspace.pact;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+import au.com.dius.pact.provider.junit5.PactVerificationContext;
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import au.com.dius.pact.provider.junitsupport.loader.PactBroker;
+import au.com.dius.pact.provider.spring.junit5.MockMvcTestTarget;
+import au.com.dius.pact.provider.spring.junit5.PactVerificationSpringProvider;
+import bio.terra.landingzone.db.LandingZoneDao;
+import bio.terra.landingzone.job.LandingZoneJobService;
+import bio.terra.policy.model.TpsPaoConflict;
+import bio.terra.policy.model.TpsPaoDescription;
+import bio.terra.policy.model.TpsPaoUpdateResult;
+import bio.terra.policy.model.TpsUpdateMode;
+import bio.terra.workspace.common.BaseAzureUnitTest;
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+import bio.terra.workspace.service.job.JobService;
+import bio.terra.workspace.service.resource.referenced.model.ReferencedResource;
+import bio.terra.workspace.service.workspace.model.Workspace;
+import bio.terra.workspace.service.workspace.model.WorkspaceStage;
+import java.util.Map;
+import java.util.UUID;
+import org.broadinstitute.dsde.workbench.client.sam.model.UserStatusInfo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.TransactionManager;
+
+@Tag("pact-verification")
+@Provider("terra-workspace-manager")
+@PactBroker()
+public class WdsContractVerificationTest extends BaseAzureUnitTest {
+  @Autowired private MockMvc mockMvc;
+
+  // this block of @MockBeans sprouted up as a result of playing whackamole with failed dependency
+  // injection errors, mostly due to NPEs through by `DataSourceInitializer`
+  @MockBean
```

Comment:
All four of these ill-advised `@MockBean`s come in due to `StartupInitializer` being run as part of the test setup.  I tried to sidestep this dependency in a number of ways (including mocking out the static method call) and wasn't able to disentangle myself from these deps.  Advice from a WSM expert would be welcome here on how not to violate the README.md's guidance to never add `@MockBean`s!

Reply:
Discussed with @blakery and these are okay to leave in.  Since this test runs in its own runtime, and isn't _really_ a unit test, it's not subject to the same concerns that the README.md had in dictating avoiding `@MockBeans`.  Resolving this one.